src/codec.ts

```diff
@@ -30,6 +30,27 @@ export interface Field {
   value: Value;
 }
 
+export interface IProtoMessageParams {
+  value: object;
+  // fully qualified name of the proto representing the enum definition
```

Comment:
Is this field only for enums? Or also for proto objects?

Reply:
This interface one is only for proto messageobjects.